problem/templates/problem/edit.html

```diff
@@ -1,108 +1,115 @@
+      </li>
+      <li>
+        <a href="#description" role="tab" data-toggle="tab">Description</a>
+      </li>
+      <li>
+        <a href="#sampio" role="tab" data-toggle="tab">Sample IO</a>
+      </li>
+      <li>
+        <a href="#tag" role="tab" data-toggle="tab">Tag</a>
+      </li>
+      <li>
+        <a href="#testcase" role="tab" data-toggle="tab">TestCase</a>
+      </li>
+    </ul>
+
+    <style>
+    .tab-pane {
+      padding-top: 10px;
+    }
+    </style>
+    <div class="tab-content">
+      <form method="POST" action='' enctype="multipart/form-data">
+        <div>
+          <a class='btn btn-success' target='_blank' id='preview_button'>Preview</a>
+          <input type="submit" value="Save" class="btn btn-primary" 
+            style="width:100px" href="{% url 'problem:problem.views.problem' %}">
+        </div>
+        {% csrf_token %}
+        <div class="tab-pane active" id="info">
+          {% if owner %}
+          <input type='hidden' name='owner' value='{{owner}}'>
+          {% endif %}
+          {% include 'problem/editTab/info.html' with form=form %}
+        </div>
+        <div class="tab-pane" id="description">
+          <strong>Description</strong>
+          <textarea cols="80" rows="20" id="des" name="description">{{description}}</textarea>
+          <br><strong>Input</strong>
+          <textarea cols="80" rows="20" id="inText" name="input_description">{{input}}</textarea>
+          <br><strong>Output</strong>
+          <textarea cols="80" rows="20" id="outText" name="output_description">{{output}}</textarea>
+
+          <script>
+          CKEDITOR.replace('des')
+          CKEDITOR.replace('inText')
+          CKEDITOR.replace('outText')
+          </script>
+        </div>
+        <div class="tab-pane" id="sampio">
+          <div class='panel panel-default' style='float:left;width:47%'>
+            <div class='panel-heading'>Sample Input</div>
+            <textarea name='sample_input' class='panel-body' 
+              style='width:100%' rows='20'>{{sample_in}}</textarea>
+          </div>
+          <div class='panel panel-default' style='float:right;width:47%'>
+            <div class='panel-heading'>Sample Output</div>
+            <textarea name='sample_output' class='panel-body' 
+              style='width:100%' rows='20'>{{sample_out}}</textarea>
+          </div>
+        </div>
+      </form>
+      <div class="tab-pane" id="tag">
+        {% include 'problem/editTab/tag.html' with pid=pid %}
+      </div>
+      <div class="tab-pane" id="testcase">
+        {% include 'problem/editTab/testcase.html' with pid=pid %}
+      </div>
+    </div>
+  </div>
+</div>
+</body>
+<script>
+$('#preview_button').click(function() {
+  MyWindow = window.open('{% url "problem:problem.views.preview" %}',
+    'MyWindow',
+    "toolbar=no,location=no,directories=no,status=no,menubar=no, \
+    scrollbars=yes,resizable=yes,width=600,height=30"
+  );
+  return false;
+});
+</script>
+{% endblock %}
```

Comment:
For overall of this file your ' and " usage may be same.
e.g. 
\<div class='panel panel-default' style='float:right;width:47%'>
and
\<div class="tab-pane" id="tag">
Their quotation usages are different.